ddpui/models/org_user.py

```
@@ -230,3 +230,48 @@ class LogoutPayload(BaseModel):
+    orguser = models.ForeignKey(OrgUser, on_delete=models.CASCADE, null=True)
+    invitation = models.ForeignKey(Invitation, on_delete=models.SET_NULL, null=True)
+    created_at = models.DateTimeField(auto_created=True, default=timezone.now)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = "orguser_group_member"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["group", "invitation"], name="uq_orguser_group_member_invitation"
+            )
+        ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)